src/test/java/org/ays/payload/RequestBodyAssignments.java

```diff
@@ -10,4 +10,23 @@ public class RequestBodyAssignments {
     private Pagination pagination;
     private FiltersForAssignments filter;
 
+    public static RequestBodyAssignments generateRequestBodyAssignments(int page, int pageSize) {
+        RequestBodyAssignments requestBodyAssignments = new RequestBodyAssignments();
+        Pagination pagination = new Pagination();
+        pagination.setPage(page);
+        pagination.setPageSize(pageSize);
+        requestBodyAssignments.setPagination(pagination);
+        return requestBodyAssignments;
+    }
```

Comment:
```suggestion
    public static RequestBodyAssignments generateRequestBodyAssignments(Pagination pagination) {
        RequestBodyAssignments requestBodyAssignments = new RequestBodyAssignments();
        requestBodyAssignments.setPagination(pagination);
        return requestBodyAssignments;
    }
```

Reply:
Bu method invalid pagination değerlerini test etmek için oluşturulmuş. Data provider ile page ve pageSize değerleri giriliyor bu nedenle iki değişken tanımlanmış. Bu şekilde kalması daha uygun olur diye düşünüyorum.

Reply:
Burayı da tekrar konuşalım